deps.bzl

```diff
@@ -4156,20 +4156,20 @@ def install_buildbuddy_dependencies(workspace_name = "buildbuddy"):
     go_repository(
         name = "io_gorm_driver_mysql",
         importpath = "gorm.io/driver/mysql",
-        sum = "h1:TATTzt+kR+IV0+h3iUB3dHUe8omCvQ0rOkmfCsUBohk=",
-        version = "v1.0.4",
+        sum = "h1:l8+9VwjjyzEkw0PNPBOr2JHhLOGVk7XEnl5hk42bcvs=",
```

Comment:
nit: can you send the gorm upgrade CL separately? I can approve it now, and it will also reveal if that upgrade alone were to break any tests.

Reply:
Yes, will do.